File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcApplicationZoomPages/AppZoom_Child2.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace C1ApplicationZoomPages
    Partial Public Class AppZoom_Child2
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub
        Private _selectIndex As Integer

        Public Property _checkParam() As Integer
            Get
                Return _selectIndex
            End Get
            Set(value As Integer)
                _selectIndex = Value
            End Set
        End Property
    End Class
End Namespace

File: NetFramework/WinForms/CS/TouchDemo/ApplicationZoomPages/AppZoom_Child2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace TouchDemo.C1ApplicationZoomPages
{
    public partial class AppZoom_Child2 : Form
    {
        public AppZoom_Child2()
        {
            InitializeComponent();
        }
        private int _selectIndex;

        public int _checkParam
        {
            get
            {
                return _selectIndex;
            }
            set
            {
                _selectIndex = value;
            }
        }

     



     
    }
}
